lib/pa_ess/updater.ex

```diff
@@ -23,20 +23,26 @@ defmodule PaEss.Updater do
         _ -> nil
       end
 
-    log_meta = [sign_id: id, current_config: log_config]
+    visual = zip_pages(top, bottom) |> format_pages()
+    tag = create_tag()
```

Comment:
Actually, a question came to mind while working on the dashboard. Is it possible to default `tag` to `nil` for legacy? It would make it easier to tell legacy and migrated messages apart and would allow me to display `N/A` in the playback column.

Reply:
Instead of that, how about another explicit field that specifies whether it's legacy? Part of the goal here is to start unifying the shape of the logs as much as possible, so I wanted to start exposing this data so we can build dashboards to that spec (but yeah, we still need a way to differentiate legacy messages)

Reply:
Yea I like that better. Much less likely to cause confusion way down the line.